.eslintrc.js

```diff
@@ -7,18 +7,20 @@ module.exports = {
     extends: [
         "plugin:vue/recommended",
         "eslint:recommended",
-        "prettier/vue",
```

Comment:
does prettier disappear?

Reply:
it`s cahnged name now. "plugin:prettier/recommended",